raspberry-pi/raspberry-pi-system.yml

```diff
@@ -0,0 +1,493 @@
+            name: Do Androids Dream of Electric Sheep?
+            type: scale
+        decimalPlaces: 1
+        height: 4
+        kind: Single_Stat_Plus_Line
+        name: Memory Usage
+        queries:
+          - query: |-
+                from(bucket: v.bucket)
+                  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+                  |> filter(fn: (r) => r["host"] == v.linux_host)
+                  |> filter(fn: (r) => r._measurement == "mem")
+                  |> filter(fn: (r) => r._field == "used_percent")
+                  |> aggregateWindow(every: v.windowPeriod, fn: mean, createEmpty: false)
+                  |> yield(name: "mean")
+        suffix: '%'
+        width: 3
+        xCol: _time
+        xPos: 9
+        yCol: _value
+        yPos: 1
+      - axes:
+          - base: "10"
+            name: x
+            scale: linear
+          - base: "2"
+            name: y
+            scale: linear
+          - base: "10"
+            name: y2
+            scale: linear
+        geom: line
+        height: 3
+        kind: Xy
+        name: Swap
+        queries:
+          - query: |-
+                from(bucket: v.bucket)
+                  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+                  |> filter(fn: (r) => r["host"] == v.linux_host)
+                  |> filter(fn: (r) => r._measurement == "swap")
+                  |> filter(fn: (r) => r._field == "total" or r._field == "used")
+                  |> aggregateWindow(every: v.windowPeriod, fn: mean, createEmpty: false)
+                  |> yield(name: "mean")
+        width: 3
+        xCol: _time
+        xPos: 9
+        yCol: _value
+        yPos: 5
+    description: A collection of useful visualizations for monitoring your system
+        stats
+    name: Raspberry Pi System
+---
+apiVersion: influxdata.com/v2alpha1
+kind: Telegraf
+metadata:
+    name: happy-wing-ecc001
+spec:
+    associations:
+      - kind: Label
+        name: tender-wu-ecc001
+    config: |
+        # Configuration for telegraf agent
+        [agent]
+          interval = "10s"
+          round_interval = true
+          metric_batch_size = 1000
+          metric_buffer_limit = 10000
+          collection_jitter = "0s"
+          flush_interval = "10s"
+          flush_jitter = "0s"
+          precision = ""
+          debug = false
+          quiet = false
+          logfile = ""
+          ## Override default hostname, if empty use os.Hostname()
+          hostname = ""
+          ## If set to true, do no set the "host" tag in the telegraf agent.
+          omit_hostname = false
+        [[outputs.influxdb_v2]]	
+          urls = ["$INFLUX_HOST"]
+          ## Token for authentication.
+          token = "$INFLUX_TOKEN"
+          ## Organization is the name of the organization you wish to write to; must exist.
+          organization = "$INFLUX_ORG"
+          ## Destination bucket to write into.
+          bucket = "telegraf"
```

Comment:
Your Flux queries use the `v.bucket` variable, but your Telegraf config is hard-coded to use `telegraf` for the bucket. If you want bucket to be user-selectable the Telegraf config should use an environment variable such as $INFLUX_BUCKET here